docs/modules/operation/pages/user-management/security-roles.adoc

```diff
@@ -10,51 +10,52 @@ endif::opennms-prime[]
 Available security roles (those with an asterisk are the most commonly used):
 
 .Functions and existing system roles in _{page-component-title}_
-[options="header, %autowidth"]
+[options="header"]
+[cols="1,3"]
 |===
 | Security Role Name  | Description
-| ROLE_ADMIN*        | Permissions to create, read, update, and delete in the web UI and the _ReST API_.
+| ROLE_ADMIN*        | Permissions to create, read, update, and delete in the web UI and the ReST API.
 | ROLE_ASSET_EDITOR | Permissions only to update the asset records from nodes.
-| ROLE_DASHBOARD    | Allow user access only to the dashboard. 
+| ROLE_DASHBOARD    | Allow user access only to the dashboard.
 | ROLE_DELEGATE     | Allow actions (such as acknowledging an alarm) to be performed on behalf of another user.
-| ROLE_FLOW_MANAGER | Allow user to edit flow classifications. 
-| ROLE_JMX          | Allow retrieving JMX metrics but does not allow executing MBeans of the {page-component-title} JVM, even if they just return simple values.
+| ROLE_FLOW_MANAGER | Allow user to edit flow classifications.
+| ROLE_JMX          | Allow retrieving JMX metrics but do not allow executing MBeans of the {page-component-title} JVM, even if they just return simple values.
 ifndef::opennms-prime[]
 | ROLE_MINION      | Minimum required permissions for a Minion to operate.
 endif::opennms-prime[]
 | ROLE_MOBILE       | Allow user to use OpenNMS COMPASS mobile application to acknowledge alarms and notifications via the REST API.
 | ROLE_PROVISION    | Allow user to use the provisioning system and configure SNMP in {page-component-title} to access management information from devices.
 | ROLE_READONLY*     | User limited to reading information in the web UI; unable to change alarm states or notifications.
-| ROLE_REPORT_DESIGNER| Permissions to manage reports in the web UI and REST API. 
+| ROLE_REPORT_DESIGNER| Permissions to manage reports in the web UI and REST API.
 | ROLE_REST         | Allow users to interact with the entire {page-component-title} REST API.
 | ROLE_RTC*          | Exchange information with the {page-component-title} Real-Time Console for availability calculations.
 | ROLE_USER*         | Default permissions for a new user to interact with the web UI: can escalate and acknowledge alarms and notifications.
 |===
 
 . Log in as a user with administrative permissions.
```

Comment:
Thoughts on this format?

```
. Log in as a user with administrative permissions.
. Click the gear icon in the top right.
. Select menu:Gear Icon[Configure OpenNMS > Configure Users, Groups and On-Call Roles > Configure Users].
. Click the modify icon next to the user you want to update.
. Select the role from menu:Security Roles[Available Roles].
. Click btn:[Add] to assign the security role to the user.
. Click btn:[Finish] to apply the changes.
. Log out and log in to apply the new security role settings.
```
![image](https://user-images.githubusercontent.com/26419758/127234339-48ab3071-e21e-4b85-93a1-fd5fbfdefece.png)

Reply:
Much more economical. Two issues: bolding the "gear icon" since the bold is supposed to represent actual text on the screen. So keep step 2, but remove "Gear Icon" from the first part of step 3. 
I also think that we can remove the "btn" syntax. We just need to put the name of the button, not typography that indicates it's a button. Will update.

Reply:
@mmahacek updated to add menu, but not button. Going forward, will update/create menus with the menu: syntax. We can use the button syntax once we update the format to be bold only without the brackets.

Reply:
Also, decided not to move the thresholding topic, as that could be part of the work in splitting up that section into smaller files. Captured here: https://issues.opennms.org/browse/NMS-13475